script.js

```diff
@@ -0,0 +1,171 @@
+  },
+  {
+    title: 'Keeping track of hundreds  of components website',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    technologies: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+    featureImg: './images/Snapshoot.svg',
+    liveLink: '#pasteLink',
+    linkSource: 'githubLink',
+    iconCancel: './images/Cancel.svg',
+  },
+  {
+    title: 'Keeping track of hundreds  of components website',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    technologies: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+    featureImg: './images/Snapshoot.svg',
+    liveLink: '#pasteLink',
+    linkSource: 'githubLink',
+    iconCancel: './images/Cancel.svg',
+  },
+  {
+    title: 'Keeping track of hundreds  of components website',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    technologies: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+    featureImg: './images/Snapshoot.svg',
+    liveLink: '#pasteLink',
+    linkSource: 'githubLink',
+    iconCancel: './images/Cancel.svg',
+  },
+];
+const seeProject = document.querySelectorAll('.popup');
+const section = document.createElement('div');
+section.className = 'popup-container';
+let content = '';
+data.forEach((item) => {
+  content = `
+  <div class="popup-details-container">
+    <span class="popup-name-image">
+      <h1 class="popup-name">${item.title}</h1>
+      <img src=${item.iconCancel} class="popup-image"> 
+    </span>
+    <ul class="popup-unorded-list">
+      <li class="popup-list">${item.technologies[0]}</li>
+      <li class="popup-list">${item.technologies[1]}</li>
+      <li class="popup-list">${item.technologies[2]}</li>
```

Comment:
- [ ] While you have done a great job, I think you should map over this array instead of using indexes, imagine in a scenario where you have more than 20 items, this will not be a good alternative.
- [ ] This is to align this part as shown in the original design
<img width="414" alt="1" src="https://user-images.githubusercontent.com/77843050/225448112-6ed7a36c-0a3c-4436-a9c7-0614c3e061ed.png">

ORIGINAL DESIGN

<img width="336" alt="2" src="https://user-images.githubusercontent.com/77843050/225448161-c84c2339-f847-4fb0-a8c5-24f2f5fec860.png">